packages/local_auth/local_auth/README.md

```diff
@@ -252,6 +252,44 @@ types (such as face scanning) and you want to support SDKs lower than Q,
 _do not_ call `getAvailableBiometrics`. Simply call `authenticate` with `biometricOnly: true`.
 This will return an error if there was no hardware available.
 
+#### Android theme
+
+You need to update the `LaunchTheme` parent style with a valid `Theme.AppCompat`
```

Comment:
This shouldn't be phrased to assume it's not already set this way, as it could be. How about:

"Your `LaunchTheme`'s parent must be a valid `Theme.AppCompat` theme to prevent crashes on Android 8 and below."